frontend/packages/topology/src/components/quick-search/QuickSearchDetails.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { history, removeQueryArgument } from '@console/internal/components/utils';
+import { CatalogItem } from '@console/plugin-sdk';
+import {
+  Button,
+  ButtonVariant,
+  Text,
+  TextContent,
+  TextVariants,
+  Title,
+} from '@patternfly/react-core';
+import './QuickSearchDetails.scss';
+
+interface QuickSearchDetailsProps {
+  selectedItem: CatalogItem;
+}
+
+const QuickSearchDetails: React.FC<QuickSearchDetailsProps> = ({ selectedItem }) => {
+  const { t } = useTranslation();
+
+  return (
+    <div className="odc-quick-search-details">
+      <Title headingLevel="h4">{selectedItem?.name}</Title>
+      {selectedItem?.provider && (
+        <span className="odc-quick-search-details__provider">
+          {t('topology~Provided by')} {selectedItem.provider}
```

Comment:
Think you're looking at doing this: https://github.com/openshift/console/blob/master/frontend/packages/dev-console/src/components/catalog/CatalogTile.tsx#L23